lib/endpoints/class-wp-json-users-controller.php

```diff
@@ -208,6 +208,9 @@ public function create_item( $request ) {
 			}
 			$user->ID = $user_id;
 		}
+
+		$this->update_additional_fields_for_object( $user, $request );
```

Comment:
Does this mean we'd return a 500 or something on an update of a post? If not, how if this error returned?